product/views.py

```diff
@@ -0,0 +1,101 @@
+import json
+from datetime            import datetime, timedelta
+
+from django.http         import JsonResponse
+from django.views        import View
+from django.db.models    import Q
+from django.utils        import timezone
+
+from .models import (
+        Product,
+        Category,
+        ProductImage,
+        ProductLike,
+        Review,
+        ReviewStatus,
+)
+
+def is_new(create_at, compare_date):
+    return True if create_at > compare_date else False
+
+def check_best_list():
+    best_query = Product.objects.all().order_by('total_sales')[:10]
+    return [best.id for best in best_query]
+
+def is_best(checkList, id):
+    return True if id in checkList else False 
+
+def is_sale(sale):
+    return True if sale > 0 else False
+
```

Comment:
```suggestion

def is_new(create_at, compare_date):
    return create_at > compare_date

def check_best_list():
    best_query = Product.objects.all().order_by('total_sales')[:10]
    return [best.id for best in best_query]

def is_best(check_list, id):
    return id in checkList

def is_sale(sale):
    return sale > 0

```
파이썬에서는 if 조건문 평가시, 반환 되는 값 자체가 boolean 일 경우나, Truthy, Falsy 한 값일 경우,  == False == None == 0등으로 평가하지 않고, 그 값을 그대로 사용합니다. 그 자체로도 True or False를 판단할 수 있기 때문입니다. False한 지 평가하실 때는 not 을 활용해보세요 :)

이것이 보다 '파이썬다운' 코드라고 할 수 있습니다